lib/render/editor/editor.html

```diff
@@ -409,23 +409,33 @@
             this._onResetState();
           })
         };
+        const _getScenarioSelector = function (scenario) {
+          return `.mu-state-${scenario} .mu-kids-state-image > *`;
+        };
+
+        const _createMultipleScenarios = () => {
+          const scenarios = ['initial', 'final'];
+          const scenarioSelectors = scenarios.map(it => _getScenarioSelector(it));
+          this.multipleScenarios = new mumuki.MultipleScenarios(scenarioSelectors);
+        };
 
         const setInitialState = () => {
-          const initialBoard = this._getTargetBoard();
-          if (!initialBoard.size) return postpone(setInitialState);
+          const initialBoards = this._getInitialBoards();
+          if (!initialBoards[0].size) return postpone(setInitialState);
 
-          this.initialState = {
-            size: initialBoard.size,
-            table: initialBoard.table,
-            header: initialBoard.header
-          };
+          this.initialState = initialBoards.toArray().map((board) => this._boardToState(board));
```

Comment:
Maybe we can just pass the `board` instance and remove the `_boardToState` method

Reply:
If we keep the reference to the board, it will be updated with the new values and we want to keep the old ones. We could copy it, but i think it's lighter to just keep the values we want